document_merge_service/conftest.py

```diff
@@ -89,3 +91,15 @@ def make_template(placeholder):
         return template
 
     return make_template
+
+
+@pytest.fixture
+def dms_test_bin():
+    sleep_path = Path(shutil.which("sleep"))
+    test_path = Path(Path(__file__).parent.absolute(), "tmpb5nw53v5")
+    with test_path.open("wb") as f:
+        with open(sleep_path, "rb") as g:
```

Comment:
You should be able to chain those `with` blocks:

```suggestion
    with test_path.open("wb") as f, open(sleep_path, "rb") as g:
        ...
```